New version: CloudFoundry.CLI.v7 version 7.7.8

SayakMukhopadhyay opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---
Closes #163401
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163415)

SayakMukhopadhyay committed:
New version: CloudFoundry.CLI.v7 version 7.7.8

stephengillie commented:
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-07-17 07:31:35.971 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\CloudFoundry.CLI.v7.7.7.8\extracted\cf.exe
> 2024-07-17 07:31:32.222 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\CloudFoundry.CLI.v7.7.7.8\extracted\cf.exe

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 886.)

SayakMukhopadhyay committed:
fix: use binary URL instead of installer URL

SayakMukhopadhyay commented:
I had added the wrong URL. Should have added the binary URL instead of the installer URL. I have fixed this in this PR and the other related PRs of this package

stephengillie commented:
- https://github.com/microsoft/winget-pkgs/issues/142854
